include/stout/flags/flags.cc

```diff
@@ -663,6 +670,52 @@ void Parser::Parse(const std::vector<ArgumentInfo>& values) {
 
 ////////////////////////////////////////////////////////////////////////
 
+void Parser::SetFieldMessageOrAggregateErrors(
+    const std::string& value,
+    const std::string& name,
+    const google::protobuf::FieldDescriptor* field,
+    google::protobuf::Message* message,
```

Comment:
can `message` be a pointer-to-const? (but first let's see if it's used, see comment below)